dffml/accuracy/accuracy.py

```diff
@@ -0,0 +1,60 @@
+import abc
+
+from ..model import ModelContext
+from ..source.source import SourcesContext
+from ..util.entrypoint import base_entry_point
+from ..base import (
+    config,
+    BaseDataFlowFacilitatorObjectContext,
+    BaseDataFlowFacilitatorObject,
+)
+
+
+class InvalidNumberOfFeaturesError(Exception):
+    pass
+
+
+@config
+class AccuracyConfig:
+    pass
+
+
+class AccuracyContext(abc.ABC, BaseDataFlowFacilitatorObjectContext):
+    def __init__(self, parent: "Accuracy") -> None:
+        self.parent = parent
+
+    @abc.abstractmethod
+    async def score(
+        self, mctx: ModelContext, sources: SourcesContext,
+    ) -> float:
+        """
+        Abstract method to get the score
+
+        Parameters
+        ----------
+        mctx : ModelContext
+            The Model which needs to be used.
+
+        sources : SourcesContext
+            The sources to use to get the accuracy
+
+        Returns
+        -------
+        float
+            The score value
+        """
+        raise NotImplementedError()
+
+
+@base_entry_point("dffml.accuracy", "accuracy")
+class AccuracyScorer(BaseDataFlowFacilitatorObject):
```

Comment:
We need to make sure that we have consistent naming. We likely should remove the `Scorer` portion of the `AccuracyScorer` class name. Since it looks like most subclasses drop the `Scorer` suffix anyway.